tests/unit/lbrynet_daemon/test_Daemon.py

```diff
@@ -269,43 +264,10 @@ def test_invalid_sort_produces_meaningful_errors(self):
             yield self.test_daemon.jsonrpc_file_list(sort=sort_options)
 
     def _get_fake_lbry_files(self):
-        return [self._get_fake_lbry_file() for _ in range(10)]
+        return list(map(lambda meta: self._get_fake_lbry_file(meta), FAKED_ATTRIBUTES))
```

Comment:
Please change to:
`[self._get_fake_lbry_file(meta) for meta in FAKED_ATTRIBUTES]`

Reply:
Done